File: exercism/python/rna-transcription/rna_transcription.py
# m = {
# 'G':'C',
# 'C':'G',
# 'T':'A',
# 'A':'U'
# }
# def to_rna(dna_strand):
    # return ''.join([m[x] for x in dna_strand])
trans = str.maketrans('GCTA','CGAU')
def to_rna(dna):
    return ''.join(dna.translate(trans))

File: exercism/typescript/rna-transcription/rna-transcription.ts
export function toRna(c: string) {
  let x = (s: string): string => {
    switch (s) {
      case "G":
        return "C";
      case "C":
        return "G";
      case "T":
        return "A";
      case "A":
        return "U";
      default:
        throw new Error("Invalid input DNA.");
    }
  };
  const r = c
    .split("")
    .map((y) => x(y))
    .join("");
  return r;
}
